azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -602,9 +604,14 @@ private void finalizeFlow(final ExecutableFlowBase flow) {
     final long durationSec = (flow.getEndTime() - flow.getStartTime()) / 1000;
     switch (flow.getStatus()) {
       case FAILED_FINISHING:
-        this.logger.info("Setting flow '" + id + "' status to FAILED in "
+        Status status = isKilled() ? Status.KILLED : Status.FAILED;
+        this.logger.info("Setting flow '" + id + "' status to "+status+" in "
```

Comment:
did you enable save plugin to auto reformat code? it should auto add new space before/after + here.